content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -118,6 +122,7 @@ These examples demonstrate the behavior of various dual-stack Service configurat
 
 #### Dual-stack options on new Services
 
+<<<<<<< HEAD
```

Comment:
This looks like a manual merge gone wrong; please fix it.

Reply:
Apologies, I must have missed this during one of the rebases I had to do.